### PR TITLE
Implementation of disable in joomla-field-switcher 

### PR DESCRIPTION
**Pull request by astridx**
See https://github.com/joomla/joomla-cms/pull/20929

### CODE REVIEW

**Comment by astridx** [comment]
Can someone help me with the conflicts, please?

File: layouts/joomla/form/field/radio/switcher.php [hunk 1]
```diff
@@ -81,7 +81,7 @@
 
 if (!empty($disabled))
 {
-	$attribs[] = 'disabled';
+	$attribs[] = 'disabled="disabled"';
```
Comment (C-Lodder) [inline layouts/joomla/form/field/radio/switcher.php]: HTML5 doesn't require `disabled="disabled"`, just `disabled`

**Comment by dgrammatiko** [comment]
@astridx you need to base your js code on: https://github.com/joomla/40-backend-template/pull/395

**Comment by astridx** [comment]
@dgrammatiko Do you mean like this: https://github.com/dgrammatiko/40-backend-template/pull/1 ?

**Comment by dgrammatiko** [comment]
Nice, php is already merged, I will marge the js this weekend. Thank you

**Pull request closed without merging**